### PR TITLE
Updated Quickstart Guide with Pip install

### PR DESCRIPTION
**Pull request by Saumya-Sunder**
Updated Quickstart Guide with Pip install

Signed-off-by: Saumya Sunder <saumya.sunder@seagate.com>

### CODE REVIEW

**Review by nileshgovande — APPROVED** [review]

File: docs/CORTX-S3 Server Quick Start Guide.md [hunk 1]
```diff
@@ -156,14 +156,14 @@ The image below illustrates the output log of a system test that is successful.
...
-1. Make sure you have installed `easy_install`
-    * To check if you have `easy_install`, run the command: `$ easy_install --version`
-    * To install `easy_install`, run the command: `$ yum install python-setuptools python-setuptools-devel`
-2. Ensure you've installed `pip`
-    * To check if you have pip installed, run the command: `$ pip --version`
-    * To install pip, run the command: `$ python --version`
-3. If you don't have Python Version 2.6.5+, then install Python using: `$ python3 --version`
-    *  If you don't have Python Version 3.3, then install python3 using: `$ easy_install pip`
+1. Make sure you have installed easy_install.
+    - To check if you have easy_install, run the command: `$ easy_install --version`
```
Comment (amitwac2608) [inline docs/CORTX-S3 Server Quick Start Guide.md:160]: we can remove the back ticks as we expect user to run this cmd on cmdline.

File: docs/CORTX-S3 Server Quick Start Guide.md [hunk 2]
```diff
@@ -156,14 +156,14 @@ The image below illustrates the output log of a system test that is successful.
...
+    - To install easy_install, run the command: `$ yum install python-setuptools python-setuptools-devel`
```
Comment (amitwac2608) [inline docs/CORTX-S3 Server Quick Start Guide.md:161]: we can remove back-ticks.

Comment (amitwac2608) [reply docs/CORTX-S3 Server Quick Start Guide.md:161]: do same for rest of the cmds in this doc.

**Review by amitwac2608 — COMMENTED** [review]
seems good. Suggested few changes

**Review by amitwac2608 — APPROVED** [review]

File: docs/CORTX-S3 Server Quick Start Guide.md (hunk 1 above)
Comment (Saumya-Sunder) [reply docs/CORTX-S3 Server Quick Start Guide.md:160]: This is used for formatting the code in .md file @amitwac2608

File: docs/CORTX-S3 Server Quick Start Guide.md (hunk 2 above)
Comment (Saumya-Sunder) [reply docs/CORTX-S3 Server Quick Start Guide.md:161]: This is used for formatting the code in .md file @amitwac2608

**Comment by nileshgovande** [comment]
Documentation PR - No need of Jenkins

**Pull request merged**